source/_integrations/workday.markdown

```diff
@@ -168,6 +168,6 @@ automation:
 
 <div class='note'>
 
-Please remember that [as explained here](/docs/configuration/devices/) you can only have a single `automation:` entry. Add the automation to your existing automations.
+Please remember that [as explained here](/docs/automation/yaml/) you should use labeled `automation` entries. You can create as many as you want.
 
 </div>
```

Comment:
This warning is indeed incorrect, and highly depending on your setup. Let just remove it to avoid confusion.